From a69a992e502fe7e46c516cebf783837ee065d77c Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 21 Nov 2022 10:55:14 +0200
Subject: [PATCH] [SW-105079] gaudi3: export security_rr_open_pcie_config api

export security_rr_open_pcie_config ROM api so that FW components
can use this api.

Change-Id: Ia5f5cc3e7b734e71e326269e16879bf054770dda
---

diff --git a/arch/gaudi3/arc/arcboot_rom.txt b/arch/gaudi3/arc/arcboot_rom.txt
index 5ac4c4b..79cd821 100644
--- a/arch/gaudi3/arc/arcboot_rom.txt
+++ b/arch/gaudi3/arc/arcboot_rom.txt
@@ -23,3 +23,4 @@
 security_get_rom_key_hash_addr
 arcboot_rom_get_spi_manifest_hdr
 arcboot_rom_get_sec_data
+security_rr_open_pcie_config
